e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/util/client/AwsS3Client.java

```diff
@@ -0,0 +1,62 @@
+/*-
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.e2e_test.util.client;
+
+import static io.restassured.RestAssured.given;
+
+import java.util.List;
+
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import lombok.SneakyThrows;
+
+import org.alfresco.hxi_connector.e2e_test.util.client.model.S3Bucket;
+import org.alfresco.hxi_connector.e2e_test.util.client.model.S3Object;
+
+public class AwsS3Client
+{
+    private final XmlMapper xmlMapper;
+    private final String baseUrl;
+
+    public AwsS3Client(String host, Integer port)
+    {
+        this.xmlMapper = new XmlMapper();
```

Comment:
you could probably do it directly on fileld

Reply:
Good hint - fixed